node-bootstrapper/app.go

```diff
@@ -100,6 +104,60 @@ func (a *App) provisionStart(ctx context.Context, cse utils.SensitiveString) err
 	return err
 }
 
+// usage example:
+// node-bootstrapper monitor
```

Comment:
can we move this to a readme instead?

Reply:
will create a followup task to add a readme